index.html

```diff
@@ -253,7 +253,8 @@ <h2 class="contact-form-title">
                         <li id="last-name"><input  maxlength = 30 type="text" placeholder="Last name" required></li>
                     </ul>
                 </li>
-                <li><input type="email"  placeholder="Email address" required></li>
+                <li><input id="email" type="email"  placeholder="Email address" required></li>
+                <span id="error" class="email-validation"></span>
                 <li><textarea name="textarea"  maxlength="500" required>Enter text here</textarea></li>
                 <li class="btn-get-in-touch"><input class="btn-green" type="submit" value="Get in touch"></li>
```

Comment:
- [ ] Form does not submit data to the formspree even when the email is provided in lowercase. Please kindly fix the submission bug